light-system-programs/scripts/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
why is there another gitignore in this subfolder?
(There is already a gitignore in light-system-programs)

Reply:
no need, removed ✅